pkgs/games/endgame-singularity/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "endgame-singularity";
+  version = "0.30c";
+  format = "other";
+
+  srcs =
+    [
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/singularity-${version}-src.tar.gz";
+        sha256 = "13zjhf67gmla67nkfpxb01rxs8j9n4hs0s4n9lnnq4zgb709yxgl";
+      })
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/endgame-singularity-music-007.zip";
+        sha256 = "0vf2qaf66jh56728pq1zbnw50yckjz6pf6c6qw6dl7vk60kkqnpb";
+      })
+    ];
+  sourceRoot = ".";
+
+  buildInputs = [ unzip ]; # The music is zipped
+  propagatedBuildInputs = with python2Packages; [ python pygame numpy ];
+
+  # Add shebang to main file
+  postPatch = ''
+    echo '#!${python2Packages.python}/bin/python'\
+         $'\n'$(cat singularity-${version}/singularity.py)\
+         > singularity-${version}/singularity.py
+  '';
+
+  # This is not an error: it needs both compilation rounds
+  buildPhase = ''
+    python -m compileall singularity-${version}
+    python -O -m compileall singularity-${version}
+  '';
+
+  installPhase = ''
+    install -Dm755 singularity-${version}/singularity.py  $out/share/singularity.py
+    install -Dm644 singularity-${version}/singularity.pyo $out/share/singularity.pyo
+    install -Dm644 singularity-${version}/singularity.pyc $out/share/singularity.pyc
+    cp -R singularity-${version}/code singularity-${version}/data $out/share/
+    cp -R endgame-singularity-music-007 $out/share/music
+  '';
+
+  # Tell it where to find python libraries
+  # Also cd to the same directory as the code, since it uses relative paths
+  postFixup = ''
+  makeWrapper "$out/share/singularity.py" "$out/bin/singularity" \
```

Comment:
Is the exe name ok or should I change it to `endgame-singularity`? Most distros and the source itself use just `singularity`

Reply:
Since you already add a wrapper here, you could just merge this with the shebang change.

I think the name is fine, especially if the official docs and other distros do the same.

Reply:
We do already have one other package with the same name: https://github.com/singularityware/singularity

But I guess conflicts can't be entirely avoided. Its your call. Changing the name from the upstream default might cause confusion.